src/.prettierrc

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
This is out of place.  `.prettierrc` should be at the project root.  Cannot accept this file.

Reply:
I didn't even see that this was there. I'll go back and review the files in this PR more carefully